projects/components/src/lib/radiobutton/radio-group.component.ts

```diff
@@ -0,0 +1,159 @@
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import {
+  AfterContentInit,
+  AfterViewInit,
+  Component,
+  ContentChildren,
+  EventEmitter,
+  Input,
+  Output,
+  QueryList,
+  forwardRef,
+} from '@angular/core';
+import '@masoni/css-radio-button/dist/index.css';
+import { MasRadioButton } from './radiobutton.component';
+import { RadioButtonClickEvent } from './radio-change';
+
+@Component({
+  selector: 'mas-radio-group',
+  template: `
+    <div class="mas-radio-group">
+      <ng-content></ng-content>
+    </div>
+  `,
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: MasRadioGroup,
+      multi: true,
+    },
+  ],
+})
+export class MasRadioGroup implements ControlValueAccessor, AfterContentInit, AfterViewInit {
+  static radioCount = 0;
+  @Output() change: EventEmitter<RadioButtonClickEvent> = new EventEmitter<RadioButtonClickEvent>();
+  @ContentChildren(forwardRef((): any => MasRadioButton)) radios?: QueryList<MasRadioButton>;
+  @Input()
+  set name(value: string) {
+    this._name = value;
+    this.updateRadios();
+  }
+  get name(): string {
+    return this._name;
+  }
+  @Input()
+  set disabled(value: boolean) {
+    this._disabled = value;
+  }
+  get disabled(): boolean {
+    return this._disabled;
+  }
+  protected _disabled: boolean = false;
+  protected _name: string = `radio-group-${MasRadioGroup.radioCount++}`;
+  protected _selected: MasRadioButton | null = null;
+  protected _value: any = null;
+  @Input()
+  set selected(selected: MasRadioButton | null) {
+    const alreadySelected = (this._selected && this._selected.value) === (selected && selected.value);
+    if (alreadySelected) {
+      return;
+    }
+    if (this._selected) {
+      this._selected.checked = false;
+    }
+    this._selected = selected;
```

Comment:
this._selected will be overwritten by selected. this._selected.checked = false will be meaningless?